include/new/trainer_sliding_data.h

```diff
@@ -14,14 +14,99 @@ extern const u8 BattleScript_TrainerSlideMsgEnd2[];
 extern const u8 gText_DefaultTrainerDynamaxMsg[];
 extern const u8 gText_TestTrainerDynamaxMsg[];
 
-//For Unbound
-
-extern const u8 sText_MirskleFirstMonDown[];
-extern const u8 sText_VegaLastSwitchIn[];
-extern const u8 sText_AliceLastSwitchIn[];
-extern const u8 sText_MelLastSwitchIn[];
-extern const u8 sText_GalavanLastSwitchIn[];
-extern const u8 sText_TessyLastSwitchIn[];
-extern const u8 sText_Zeph1LowHP[];
-extern const u8 sText_Zeph2LowHP[];
-extern const u8 sText_MaximaLastSwitchIn[];
+// Rival battle 1
+extern const u8 sText_Rival1_LastSwitchIn[];
+extern const u8 sText_Rival1_LowHP[];
+
+// Rival battle n
+
+// Gym Leader 1 (Terrence)
+extern const u8 sText_Leader1_FirstMonDown[];
+extern const u8 sText_Leader1_LastSwitchIn[];
+extern const u8 sText_Leader1_LowHP[];
+
+// Gym Leader 2 (Stacy)
+extern const u8 sText_Leader2_FirstMonDown[];
+extern const u8 sText_Leader2_LastSwitchIn[];
+extern const u8 sText_Leader2_LowHP[];
+
+// Gym Leader 3 (Wendy)
+extern const u8 sText_Leader3_FirstMonDown[];
+extern const u8 sText_Leader3_LastSwitchIn[];
+extern const u8 sText_Leader3_LowHP[];
+
+// Gym Leader 4 (Monty)
+extern const u8 sText_Leader4_FirstMonDown[];
+extern const u8 sText_Leader4_LastSwitchIn[];
+extern const u8 sText_Leader4_LowHP[];
+
+// Gym Leader 5 (Chandler)
+extern const u8 sText_Leader5_FirstMonDown[];
+extern const u8 sText_Leader5_LastSwitchIn[];
+extern const u8 sText_Leader5_LowHP[];
+
+// Gym Leader 6 (Abby)
+extern const u8 sText_Leader6_FirstMonDown[];
+extern const u8 sText_Leader6_LastSwitchIn[];
+extern const u8 sText_Leader6_LowHP[];
+
+// Gym Leader 7 (Iris)
+extern const u8 sText_Leader7_FirstMonDown[];
+extern const u8 sText_Leader7_LastSwitchIn[];
+extern const u8 sText_Leader7_LowHP[];
+
+// Gym Leader 8 (Dennis & Dee)
+extern const u8 sText_Leader8_FirstMonDown[];
+extern const u8 sText_Leader8_LastSwitchIn[];
+extern const u8 sText_Leader8_LowHP[];
+
+// Team Pluto Admin Irene
+extern const u8 sText_Irene_FirstMonDown[];
```

Comment:
some areas you are doing things like
extern const u8 sText_Leader1_FirstMonDown[];  
(note how the name of the leader is not included)

when you are refering to a particular person in other places you are doing 
extern const u8 sText_Irene_FirstMonDown[]; 

my suggestion is to be consistent with the naming.
extern const u8 sText_PlutoAdmin_Irene_FirstMonDown[]; 
extern const u8 sText_GymLeader_Terrence_FirstMonDown[];  
..
or 
extern const u8 sText_Leader1_FirstMonDown[];  
extern const u8 sText_PlutoAdmin1_FirstMonDown[]; 
extern const u8 sText_PlutoLeader_FirstMonDown[]; 

so tdlr: drop the name all together or include it in places you have not.  this change will have to get done in many places

Reply:
I'm partial to keeping it this way, at least for now, to slim up on characters.  IMO the trainer name (or # for something more regular like gym leader order) is clear enough to read.  LMK what you think.